package.json

```diff
@@ -51,7 +51,8 @@
     "peer-info": "^0.7.1",
     "pre-commit": "^1.1.3",
     "run-parallel": "^1.1.6",
-    "run-series": "^1.1.4"
+    "run-series": "^1.1.4",
+    "webpack": "2.1.0-beta.25"
```

Comment:
`aegir`should be the one bringing webpack. if beta versions are making it fail, then we need to lock it to one that does make our modules break. 

If this is a problem here, it is a problem for everything, @dignifiedquire can you confirm?